lib/modules/dosomething/dosomething_mbp/dosomething_mbp.install

```diff
@@ -135,9 +135,68 @@ function dosomething_mbp_update_7007() {
   dosomething_mbp_install_productions();
 }
 
+/**
+ * Add config for Mailsystem and Mandrill modules.
+ */
+function dosomething_mbp_update_7008() {
+  module_enable('mailsystem', 'mandrill');
+  $mailsystem_keys = array(
```

Comment:
Do we need to have these settings set in the install hook as well?

Reply:
I was basing this on the assumption that this module would never be installed again (on a new site or an environment where it's not already installed), as opposed to updated. Is that not a safe assumption?

Reply:
Yeah, I think that's a valid assumption - just wanted to ask and see what you thought / if it was unnecessary.